org-formation/_scp.yaml

```diff
@@ -126,3 +161,19 @@
               - 'route53domains:RegisterDomain'
               - 'route53domains:TransferDomain'
             Resource: '*'
+
+  # from https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_scps_examples_general.html#example-scp-deny-region
+  RestrictLeaveOrganizationSCP:
+    Type: OC::ORG::ServiceControlPolicy
+    Properties:
+      PolicyName: RestrictLeaveOrganization
+      Description: Administrators of member accounts can't remove their accounts from the organization
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: DenyLeaveOrganization
+            Effect: Deny
+            Action:
+              - 'route53domains:RegisterDomain'
```

Comment:
Should this not be 'organizations:LeaveOrganization'  ?

Reply:
yes, of course.  fixed!